src/saveload/saveload.h

```diff
@@ -15,6 +15,278 @@
+	SLV_164,                                // 164   23290
+
+	SLV_165,                                // 165   23304
+	SLV_166,                                // 166   23415
+	SLV_167,                                // 167   23504
+	SLV_168,                                // 168   23637
+	SLV_169,                                // 169   23816
+
+	SLV_170,                                // 170   23826
+	SLV_171,                                // 171   23835
+	SLV_172,                                // 172   23947
+	SLV_173,                                // 173   23967   1.2.0-RC1
+	SLV_174,                                // 174   23973   1.2.x
+
+	SLV_175,                                // 175   24136
+	SLV_176,                                // 176   24446
+	SLV_177,                                // 177   24619
+	SLV_178,                                // 178   24789
+	SLV_179,                                // 179   24810
+
+	SLV_180,                                // 180   24998   1.3.x
+	SLV_181,                                // 181   25012
+	SLV_182,                                // 182   25115 FS#5492, r25259, r25296 Goal status
+	SLV_183,                                // 183   25363 Cargodist
+	SLV_184,                                // 184   25508 Unit localisation split
+
+	SLV_185,                                // 185   25620 Storybooks
+	SLV_186,                                // 186   25833 Objects storage
+	SLV_187,                                // 187   25899 Linkgraph - restricted flows
+	SLV_188,                                // 188   26169 FS#5831 Unify RV travel time
+	SLV_189,                                // 189   26450 Heirarchical vehicle subgroups
+
+	SLV_190,                                // 190   26547 Separate order travel and wait times
+	SLV_191,                                // 191   26636 FS#6026 Fix disaster vehicle storage (No bump)
+	                                        // 191   26646 FS#6041 Linkgraph - store locations
+	SLV_192,                                // 192   26700 FS#6066 Fix saving of order backups
+	SLV_193,                                // 193   26802
+	SLV_194,                                // 194   26881 v1.5
+
+	SLV_195,                                // 195   27572 v1.6.1
+	SLV_196,                                // 196   27778 v1.7
+	SLV_197,                                // 197   27978 v1.8
+	SLV_198,                                // 198  PR#6763 Switch town growth rate and counter to actual game ticks
+	SLV_199,                                // 199  PR#6802 Extend cargotypes to 64
+
+	SLV_200,                                // 200  PR#6805 Extend railtypes to 64, adding uint16 to map array.
+	SLV_201,                                // 201  PR#6885 Extend NewGRF persistant storages.
+	SLV_202,                                // 202  PR#6867 Increase industry cargo slots to 16 in, 16 out
+	SLV_203,                                // 203  PR#7072 Add path cache for ships
```

Comment:
Is it possible to give last saveloads a more descriptive name than SLV_203? Let's say SLV_203_SHIP_CACHE?
It may be helpful to have them as an example for future naming of saveload versions.

Reply:
Yes, the intention is to not include the number, however, as that would kinda defeat the point.

Reply:
Just last ones, to make clear future developers how to give proper names to saveload versions.

Reply:
I would keep saveload number in the name for merged commits.

Reply:
Hmm, that may end up with future additions following suit, when it's unnecessary and creates more work.

Reply:
The numbers on the saveload names may be helpful if we find a bug on an old savegame. Knowing which pieces of code are executed and which ones aren't on AfterLoad() is important.

Reply:
I disagree, the number should not be part of (future) saveload enum entries, because the whole point of the exercise is to make it easier to maintain patches with savegame bumps, who do not know yet what the final number will be.